python/flatbuffers/number_types.py

```diff
@@ -170,3 +172,11 @@ def uint64_to_float64(n):
     packed = struct.pack("<1Q", n)
     (unpacked,) = struct.unpack("<1d", packed)
     return unpacked
+
+
+def to_numpy_type(number_type):
+    if np is not None:
+        return np.dtype(number_type.name).newbyteorder('<')
+    else:
+        raise RuntimeError(('Numpy could not be imported'
```

Comment:
Sorry, I don't understand. Do you want error in a function to reduce duplicated code in number_types.py and encode.py?